src/CMakeLists.txt

```diff
@@ -26,34 +26,72 @@ set(SOURCES
     utils/roundrobin_poll_strategy.cpp
 )
 
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../include/cppkafka)
-include_directories(SYSTEM ${Boost_INCLUDE_DIRS} ${RDKAFKA_INCLUDE_DIR})
+set(TARGET_NAME         cppkafka)
+set(PKG_DIR             "${PROJECT_SOURCE_DIR}/package")
```

Comment:
Can we put this inside the build directory so it doesn't pollute the source one?

Reply:
yeah ok.